src/Filters/Company.jsx

```diff
@@ -0,0 +1,104 @@
+import { addMultipleFilters } from '../actions/filter'
+import { bindAll } from '../utils'
+import CollapsibleFilter from './CollapsibleFilter'
+import { connect } from 'react-redux'
+import HighlightingOption from '../Typeahead/HighlightingOption'
+import PropTypes from 'prop-types'
+import React from 'react'
+import StickyOptions from './StickyOptions'
+import Typeahead from '../Typeahead'
+
+const FIELD_NAME = 'company'
+
+export class Company extends React.Component {
+  constructor( props ) {
+    super( props )
+
+    // Bindings
+    bindAll( this, [
+      '_onInputChange',
+      '_onOptionSelected',
+      '_renderOption'
+    ] )
+  }
+
+  render() {
+    const desc = 'The company name as it appears in our complaint system, ' +
+      'which may be different than the name the consumer provided in their ' +
+      'complaint'
+
+    return (
+      <CollapsibleFilter title="Matched Company Name"
```

Comment:
The name should be sentence case. "Matched company name"

Reply:
Good catch! Will fix.